cmd/konnectd/serve.go

```diff
@@ -158,8 +166,12 @@ func serve(cmd *cobra.Command, args []string) error {
 
 	// Survey support.
 	var guid []byte
-	if bs.issuerIdentifierURI.Hostname() != "localhost" {
-		guid = []byte(bs.issuerIdentifierURI.String())
+	issUrl, err := url.Parse(bootstrapConfig.ISS)
```

Comment:
I think this value should be parsed somewhere else (in bootstrap).

Reply:
This would couple the survey-client to the bootstrap. Library consumers might not want to use/include the survey client?